.circleci/config.yml

```diff
@@ -889,6 +897,13 @@ commands:
     steps:
       - run: echo "export CLUSTER_ID=\"<<parameters.cluster-id>>\"" >> $BASH_ENV
 
+  set-cluster-labels:
+    parameters:
+      labels:
+        type: string
+    steps:
+      - run: echo "export LABELS=\"$(echo '<<parameters.labels>>' | sed 's/ /-/g')\"" >> $BASH_ENV
```

Comment:
Man, I can't believe you're skipping a perfectly good opportunity to use `tr` *and* `printf`. ;)